packages/cli/src/PublicApi/v1/handlers/workflows/workflows.service.ts

```diff
@@ -98,14 +62,14 @@ export async function createWorkflow(
 	});
 }
 
-export async function setWorkflowAsActive(workflow: WorkflowEntity): Promise<UpdateResult> {
+export async function setWorkflowAsActive(workflow: WorkflowEntity) {
 	return Container.get(WorkflowRepository).update(workflow.id, {
```

Comment:
since these results aren't actually used anywhere, perhaps we should replace `return` with an `await` instead ?